config/storage/key-value/resource-store.json

```diff
@@ -56,6 +56,19 @@
           "PathBasedAuthorizer:_paths_value": { "@type": "DenyAllAuthorizer" }
         }
       ]
+    },
+    {
+      "comment": "Marks the /.internal/ storage container as an auxiliary resource, thereby hiding it from container representations.",
+      "@id": "urn:solid-server:default:AuxiliaryStrategy",
+      "RoutingAuxiliaryStrategy:_sources": [
+        {
+          "@type": "ComposedAuxiliaryStrategy",
```

Comment:
Why do se need a composed wrapper here?

Reply:
Because the `SuffixAuxiliaryIdentifierStrategy` is an `AuxiliaryIdentifierStrategy` and the `ComposedAuxiliaryStrategy` is an `AuxiliaryStrategy`. We need the second one. None of the extra features that can be added are used here, but we still need the function calls that are provided by that class.